### ISSUE TITLE
Change analysis plot x and y labels and buttons from acronymns

### ISSUE THREAD

**Issue by k-shep** [SABS-Group-2-2021-22/drug-discovery-game-app#149]

**Comment by guydurant** [comment]
I think your fix requires a bit more work, currently the graph is not working

**Comment by guydurant** [comment]
@k-shep Yes that broke the ability for the graph to recognise which thing to plot unfortunately - we need to find a way to change the label of the axes without affecting the graph building

**Comment by k-shep** [comment]
Oh, sorry about that. I didn't realise.

**Comment by guydurant** [comment]
Ive found a fix dw

**Comment by k-shep** [comment]
Thank you and thanks for spotting it!

**Comment by guydurant** [comment]
Ive pushed to main about the axes, if you want to fix other buttons, follow the method i did to fix it
